examples/python/cli/cli.py

```diff
@@ -44,19 +44,27 @@ def is_paid(self, destination: str):
     def is_synced(self):
         return self.listener.is_synced()
     
+    def generate_mnemonic(self):
+        mnemo = Mnemonic("english")
+        words = mnemo.generate(strength=128)
+        return words
+
     def read_mnemonic(self):
         if exists('phrase'):
             with open('phrase') as f:
                 mnemonic = f.readline()
+                if not mnemonic:
```

Comment:
If the mnemonic, for any reason, is not in the first line (e.g. due to user error) it will be overwritten.